src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -509,7 +563,20 @@ private static bool ComputeNeedsValueContentAnalysis(IOperation operationBlock,
                         var parentAttributes = CopyAttributes(attributes);
                         using var parentCapturedVersions = PooledDictionary<string, Version>.GetInstance(capturedVersions);
 
-                        if (!IsKnownValueGuarded(parentAttributes, ref csAttributes, parent, parentCapturedVersions))
+                        if (value.AnalysisValues.Count != 0 && value.AnalysisValues.Count == parent.AnalysisValues.Count)
+                        {
+                            var parentEnumerator = parent.AnalysisValues.GetEnumerator();
+                            foreach (var val in value.AnalysisValues)
+                            {
+                                if (parentEnumerator.MoveNext() && !parentEnumerator.Current.Equals(val.GetNegatedValue()))
+                                {
+                                    goto label;
```

Comment:
Suggestion: Use `break` instead of `goto`.

Reply:
No, it was breaking the outer loop, anyway used a function instead